search/documents.py

```diff
@@ -0,0 +1,17 @@
+from django_elasticsearch_dsl import Document
+from django_elasticsearch_dsl.registries import registry
+from foods.models.FoodModel import Food
+
+
+@registry.register_document
+class FoodDocument(Document):
+    class Index:
+        name = 'foods'
+        settings = {
+            'number_of_shards': 1,
```

Comment:
number_of_shards là gì?

Reply:
shrink selected indices on the node with the most available space.

Reply:
Nó dùng tính toán score cho (from + size) bản ghi trên mỗi shard, theo công thứ number_of_shards * (from + size).